composer.json

```diff
@@ -11,13 +11,18 @@
         "webmozart/assert": "1.4"
     },
     "require-dev": {
-        "guzzlehttp/guzzle": "~6.0",
         "phpunit/phpunit": "~5.7",
-        "simplesamlphp/simplesamlphp": "^1.17",
+        "simplesamlphp/simplesamlphp": "dev-master",
         "satooshi/php-coveralls": "^1.0"
     },
     "support": {
         "issues": "https://github.com/simplesamlphp/simplesamlphp-module-casserver/issues",
         "source": "https://github.com/simplesamlphp/simplesamlphp-module-casserver"
+    },
+    "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/simplesamlphp/simplesamlphp"
     }
```

Comment:
I don't understand this change.. Should just be ^1.17?

Reply:
Without using the repo and the master branch, simplesamlphp checks out without the tests folder. And that's my main problem.

Reply:
No, that can't be..  You must be doing something unconventional

Reply:
Could it be that your IDE is messing with us here?  A composer-install of this module will install simplesamlphp in the vendor-dir including the tests-dir...

Maybe we can fix this as follows in composer.json:
https://github.com/tvdijen/simplesamlphp-module-monitor/blob/a856f6d81e0df7dfb098e9ee830b9be022b9f6be/composer.json#L13-L18